src/actors/MeleeMob.gd

```diff
@@ -31,9 +38,9 @@ var _dead = false
 var _drop_loot = true
 
 
-
 func _ready():
-	target = sigil
+	movement_target = sigil
+	player = get_parent().get_parent().get_node('player')
```

Comment:
This will break if the tree structure changes at all, we should look into grabbing the player from a group, or via the root.

Reply:
niiiiice fixed